xml/System.Security.Cryptography/SHA256.xml

```diff
@@ -112,7 +110,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Creates an instance of the default implementation of <see cref="T:System.Security.Cryptography.SHA256" />.</summary>
+        <summary>Creates an instance of the default implementation of <see cref="T:System.Security.Cryptography.SHA256" />. When this behavior is not overriden, the default implementation is <see cref="T:System.Security.Cryptography.SHA256Managed" />.</summary>
```

Comment:
On .NET Framework the default return type is SHA256Managed unless FIPS mode is active, then it uses SHA256Cng.

On .NET Core the behavior cannot be altered and is an opaque type.  (It returns an instance of a private class)

Reply:
@bartonjs Great information, thanks ! Do you have any pointer about this specific behavior implementation in .NET Framework ? The only related thing I found in referencesource.microsoft.com ([here](http://referencesource.microsoft.com/#System.ServiceModel/System/ServiceModel/Security/CryptoHelper.cs,6ba0c31c543fc455,references)) is located in System.ServiceModel and returns a SHA256CryptoServiceProvider in FIPS mode...

@rpetrusha What would be the right way to show the difference between .NET Framework and .NET Core behaviors ?

Reply:
http://referencesource.microsoft.com/#mscorlib/system/security/cryptography/cryptoconfig.cs,238 (lines 237-248)

Reply:
@bartonjs Thanks a lot for this ! I don't even understand how I missed it

Reply:
@sputier, it's best to remove this from the summary section and add it to the \<returns> section. It can then read:

\<returns>A new instance of \<see cref="T:System.Security.Cryptography.SHA256"/>. On the .NET Framework, this method creates an instance of the \<see cref="T:System.Security.Cryptography.SHA256Managed/> class if FIPS mode is not active; if FIPS mode is active, it creates an instance of the \<see cref="System.Security.Cryptography.SHA256Cng"/> class. On .NET Core, it returns an instance of a private class derived from \<see cref="System.Security.Cryptography.SHA256"/>.

Reply:
@rpetrusha Thanks ! I integrate your suggestion in the next commit.